libs/ledger-live-common/src/families/bitcoin/wallet-btc/explorer/index.ts

```diff
@@ -15,8 +15,7 @@ type ExplorerParams = {
 };
 
 class BitcoinLikeExplorer implements IExplorer {
-  client: Pool<{ client: AxiosInstance }>;
-  underlyingClient: AxiosInstance;
+  client: AxiosInstance;
```

Comment:
You can even try using our `network` module instead of `axios`, to be more consistent with our other integrations.

Reply:
fixed